infra/modules/providers/azure/cosmosdb/output.tf

```diff
@@ -0,0 +1,9 @@
+output "cosmosdb_endpoint" {
+  description = "The endpoint used to connect to the CosmosDB account."
+  value       = "${azurerm_cosmosdb_account.cosmosdb.endpoint}"
+}
+
+output "cosmosdb_primary_master_key" {
+    description = "The Primary master key for the CosmosDB Account."
+    value = "${azurerm_cosmosdb_account.cosmosdb.primary_master_key}"
```

Comment:
this output should be marked as `sensitive = true`

Reply:
@erikschlegel Done, pushed the updated output to the PR.